src/content/learn/preserving-and-resetting-state.md

```diff
@@ -561,11 +561,11 @@ label {
 
 </Sandpack>
 
-Here, you switch between _different_ component types at the same position. Initially, the first child of the `<div>` contained a `Counter`. But when you swapped in a `p`, React removed the `Counter` from the UI tree and destroyed its state.
+Here, you switch between _different_ Component types at the same position. Initially, the first child of the `<div>` contained a `Counter`. But when you swapped in a `p`, React removed the `Counter` from the UI tree and destroyed its state.
```

Comment:
This one feels tricky @rickhanlonii do you think this should be Component or component ? Since we are talking about the react context here but also referencing one React Component (Counter) but also built ins (<p>) ?